Themes/default/Reports.template.php

```diff
@@ -111,12 +111,12 @@ function template_main()
 				// Shaded?
 				if ($column_number == 0 && !empty($table['shading']['left']))
 					echo '
-					<td align="', $table['align']['shaded'], '" class="table_caption"', $table['width']['shaded'] != 'auto' ? ' width="' . $table['width']['shaded'] . '"' : '', '>
```

Comment:
Since we're setting the alignment based on a variable, we should handle that accordingly rather than assuming they'll always be the default values. Just set `class="', $table['align']['shaded'], 'text"` or something similar.